src/module-1/weirdString/index.js

```diff
@@ -1 +1,15 @@
-export const weirdString = (str = "") => {};
+export const weirdString = (str = "") => {
+	let result = "";
+	let arr = str.split(" ");
+	for (let i = 0; i < arr.length; i++) {
```

Comment:
would be nice if you used `join` and  `map`.

Reply:
fixed

Reply:
I used `join`.